App.py

```
@@ -0,0 +1,137 @@
+import streamlit as st
+import streamlit as st
+import sqlite3
+from google import genai as genai2
+from PyPDF2 import PdfReader
```

Ruff (0.12.2)

`PyPDF2.PdfReader` imported but unused

Remove unused import: `PyPDF2.PdfReader`

(F401)